sdcm/nemesis.py

```diff
@@ -156,6 +156,8 @@ class Nemesis:  # pylint: disable=too-many-instance-attributes,too-many-public-m
     kubernetes: bool = False        # flag that signal that nemesis run with k8s cluster
     limited: bool = False           # flag that signal that nemesis are belong to limited set of nemesises
     has_steady_run: bool = False    # flag that signal that nemesis should be run with perf tests with steady run
+    tombstone_gc_mode: bool = False  # A flag denotes a nemesis impacts tombstone and its GC,
```

Comment:
@fgelcer , please note:
1. There is a very clear and comprehensive comment, explaining what this flag is about.
2. Since it is a `flag`, of course it is a boolean. 
3. It just follows the convention of all upper flags.
4. The basic idea is for it to be a property name, so any name is good enough i think. so for every such property, then nemesis set whether it belongs to it or not, by the boolean setting.
5. Anyway if you have an idea for a better name please suggest. the idea is to flag if the nemesis has an impact on tombstones or not. so how about renaming it to something like "tombstone_impact" ?

Reply:
> @fgelcer , please note:
> 
> 1. There is a very clear and comprehensive comment, explaining what this flag is about.

the comment is here only, not in other places where this is being used... and the comment is not clear, for me at least

> 2. Since it is a `flag`, of course it is a boolean.

if this is a flag, it shall be more clear... remember that, `gc mode` is not clear to everyone... use a better naming, please

> 3. It just follows the convention of all upper flags.

it does, but it has a bad name...

> 4. The basic idea is for it to be a property name, so any name is good enough i think. so for every such property, then nemesis set whether it belongs to it or not, by the boolean setting.

it is fine to be a boolean, and it is fine the way it is defined, but name should be improved...

> 5. Anyway if you have an idea for a better name please suggest. the idea is to flag if the nemesis has an impact on tombstones or not. so how about renaming it to something like "tombstone_impact" ?

`changes_tombstone_gc_mode` may be a bit better... but feel free to improve it to other name, if you prefer

Reply:
`changes_tombstone_gc_mode` is not correct for this set of nemeses. `soft reboot`, for example, doesn't change gc mode.
Let's go with "tombstone_impact"